menu.h

```diff
@@ -200,6 +242,86 @@ class Menu {
             draw_overlay(button->config);
     }
 
+        // Method to update the sprite positions and draw sprite.
+    void update_slide(sprite button_sprite, int position)
+    {
+        // Show the base layer of sprite.

```

Comment:
To solve the overlay issue, we could just turn the overlay off when sliding. eg:
`this->_overlayActive = false;`

Reply:
Thanks Anthony, I have now added this.